src/components/FormOfSeach/FormFieldSearchHotel/FormFieldSearchHotel.js

```diff
@@ -1,46 +1,67 @@
 import React, {useEffect, useState} from "react";
-import './FormFieldSearchHotel.css'
+import {useDispatch, useSelector} from "react-redux";
+import {inputText} from "../../../redux/actions";
+import {
+    StyledInputFist,
+    StyledInputWrapper,
+    StyledLabelFist
+} from "../../Styled-components/HeaderForm";
 
-export const useForm = () => {
-    const [values, setValues] = useState({
-        nameOfSearch: '',
-    });
-
-    const handleChange = (e) => {
-        const {name: inputName} = e.target;
-
-
-        setValues({
-            ...values,
-            [inputName]: e.target.value,
-        });
+const FirstForm = () => {
+    const [values, setValues] = useState('');
+    const [isActive, setIsActive] = useState(false);
+    const handleFocus = (e) => {
+        if (e.target.value === "") {
+            disableField(e);
+        }
+        setIsActive(true);
+    };
+    const disableField = () => {
+        setIsActive(false);
+    };
+    const disableFocus = (e) => {
+        if (e.target.value === "") {
+            disableField(e);
+        }
+    };
+    const dispatch = useDispatch();
+    const handleChange = (value) => {
+        dispatch(inputText(value))
     }
+    const searchText = useSelector(state => state.formReducer.search);
+    useEffect(
+        () => {
+            if (searchText.length !== 0 && (window.scrollY > 2400)) {
+                setIsActive(true)
+                window.scrollTo({
```

Comment:
this is the reason why I asked to move into hook - you use it in multiple places